search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -98,3 +98,95 @@ function displayTopResultsForCurrentTrend(countryCode) {
   }
   resultElement.innerHTML += '<i>Last updated on ' + date + '</i>';
 }
+
+/**
+ * Sets trends under the top-trends modal tab for the selected country.
+ * @param {string} countryCode The two-letter code of the selected country.
+ */
+function setCountryTrends(countryCode) {
+  const topTrendsTab = document.getElementById('top-trends-tab');
+  topTrendsTab.innerHTML = '';
+  fetch('/country-trends/' + countryCode).then(countryTrends =>
+      countryTrends.json()).then(trends => {
+        if (trends.length === 0) {
+          topTrendsTab.innerText = 
+              'Trends are not available for the selected country.';
+        } else {
+          trends.forEach(trend => {
+            const trendHeader = document.createElement('h5');
+            trendHeader.innerText = trend.topic;
+            topTrendsTab.appendChild(trendHeader);
+          });
+        }
+      });
+}
+
+/** 
+ * Displays the popularity timeline in a country for the current search topic 
+ * in the modal.
+ * @param {string} countryCode The two-letter code of the selected country.
+ */
+function displayPopularityTimeline(countryCode) {
+  const popTimelineElement = document.getElementById('pop-timeline-tab');
+  popTimelineElement.innerHTML = '';
+  fetch('/trends/', {
+    method: 'post',
+    headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      topic: getCurrentSearchData().topic,
+      code: countryCode,
+    })
+  })
+  .then(interestData => interestData.json())
+  .then(timelineJSON => {
+      if (timelineJSON.length === 0) {
+        popTimelineElement.innerText = 
+            'Popularity Timeline is not available for the selected country.';
+      } else {
+        drawPopularityTimeline(timelineJSON.default, popTimelineElement, 
+            getCurrentSearchData().topic);
+      }
+    });
+}
+
+// Use version 45 to allow for chart ticks to be drawn when the div container is
+// hidden.
+google.charts.load('45', {'packages':['corechart']});
+/** 
+ * Draw the popularity timeline in a country for the current search trend on
+ * modal.
+ * @param {!Array<JSON>} 10 globally trending topics and the number of countries
```

Comment:
```suggestion
 * @param {!Array<JSON>} timelineJSON 10 globally trending topics and the number of countries
```
Also I don't actually understand what this param holds. Try and rephrase this comment better to make this clearer.